Source/JavaScriptCore/llint/WebAssembly.asm

```diff
@@ -659,37 +628,56 @@ end
         end
     end
 
-    clobberVolatileRegisters()
-
+    macro repeat(f)
+        move 0xBEEF, ws0
+        f(0)
+        f(1)
+        f(2)
+        f(3)
+        f(4)
+        f(5)
+        f(6)
+        f(7)
+        f(8)
+        f(9)
+        f(10)
+        f(11)
+        f(12)
+        f(13)
+        f(14)
+        f(15)
+        f(16)
+        f(17)
+        f(18)
+        f(19)
+        f(20)
+        f(21)
+        f(22)
+        f(23)
+        f(24)
+        f(25)
+        f(26)
+        f(27)
+        f(28)
+        f(29)
+    end
 
-if ARM64 or ARM64E or ARMv7
-    const JSEntrypointInterpreterSpillSpaceStackAligned = (4 * SlotSize + StackAlignment - 1) & ~StackAlignmentMask
-else
-    const JSEntrypointInterpreterSpillSpaceStackAligned = (8 * SlotSize + StackAlignment - 1) & ~StackAlignmentMask
+if ASSERT_ENABLED or true
```

Comment:
Presumably the `or true` was temporary here (and four other times below)?

Reply:
Yeah